modules/zram/udiskslinuxmanagerzram.c

```diff
@@ -204,7 +204,8 @@ create_conf_files (guint64   num_devices,
   guint64 i;
 
   filename = g_build_filename (PACKAGE_MODLOAD_DIR, "zram.conf", NULL);
-  contents = g_strdup ("zram\n");
+  contents = g_strdup ("# UDisks2 managed ZRAM configuration\n"
```

Comment:
Maybe also add a line about this file being automatically created by UDisks and changes will be overwritten.

Reply:
Good idea, added a note.